src/app/components/Input/InputGroup.tsx

```diff
@@ -6,12 +6,12 @@ const InputAddon = styled.div`
 	${tw`absolute inset-y-0 flex items-center justify-center overflow-hidden`};
 `;
 
-export const InputAddonStart = styled(InputAddon)`
-	${tw`left-0 rounded-l border border-transparent`}
+export const InputAddonStart = styled.div`
+	${tw`border border-r-0 rounded-l`}
```

Comment:
`InputAddonStart` and `InputAddonEnd` have the same purpose, it is just to position it absolute within a relative parent